go/systests/team_tx_test.go

```diff
@@ -149,7 +149,7 @@ func TestTeamTxDependency(t *testing.T) {
 
 	tx := teams.CreateAddMemberTx(teamObj)
 	tx.AddMemberByUsername(context.Background(), tracy.username, keybase1.TeamRole_READER)
-	tx.AddMemberByUsername(context.Background(), bob.username, keybase1.TeamRole_WRITER)
+	tx.AddMemberByUsername(context.Background(), bob.username, keybase1.TeamRole_ADMIN)
```

Comment:
@heronhaye maybe also add the new thing you're testing here (high info) to the description above, since it's not really obvious that this test is now also testing something else.